src/main.rs

```diff
@@ -327,6 +327,7 @@ fn run() -> Result<()> {
             unix::upgrade_gnome_extensions(&ctx)
         })?;
         runner.execute(Step::Pyenv, "pyenv", || unix::run_pyenv(&ctx))?;
+        runner.execute(Step::Uv, "uv", || unix::run_uv(&ctx))?;
```

Comment:
Looks like this tool is also available on Windows, we should probably move the impl to `general.rs`👀